docs/DeveloperGuide.adoc

```diff
@@ -1064,42 +1057,38 @@ Certain properties of the application can be controlled (e.g user prefs file loc
 Different view has its own designated filter command.
 
 ====
-* Student View, `Student filter` command is provided.
-* Session View, `Session filter` command is provided.
-* Claims View, `Claims filter` command is provided.
+* Student View, `student filter` command is provided.
+* Session View, `session filter` command is provided.
+* Claims View, `claims filter` command is provided.
```

Comment:
i think this whole section for filtering should be categorised under student, session and claims rather than it being an entire section on its own. will make a mental note to ask yijing about this later

Reply:
Okie! But i think a separate section also can. That way it'll be easier for her to mention it in her PPP!